### PR TITLE
use MatchesSelector to implement simple localized querySelector

### PR DESCRIPTION
**Pull request by sjmiles**
Arv, PTAL, I don't know if this is the way you would like to do this.

References #70.

### CODE REVIEW

**Comment by sjmiles** [comment]
Thanks for all the notes Arv, Please look at the latest revision when you get a chance.

File: src/querySelector.js [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+  'use strict';
+
+  // imports
+   
+  var unwrap = scope.unwrap;
+
+  // find a MatchesSelector for this platform
+    
+  // spec
+  var matches = Element.prototype.matches;
```
Comment (arv) [inline src/querySelector.js:15]: There is also:

https://github.com/toolkitchen/ShadowDOM/blob/master/src/ShadowRenderer.js#L225

We should get rid of one of them.

**Comment by arv** [comment]
LGTM

**Comment by dfreedm** [comment]
LGTM2

**Comment by arv** [comment]
Fixed in 3373192d44e2b0da37b8d9d2dd006a631fa061ed

**Pull request closed without merging**